README.md

```diff
@@ -3,9 +3,10 @@ cgm-remote-monitor (a.k.a. Nightscout)
 
 [![Build Status][build-img]][build-url]
 [![Dependency Status][dependency-img]][dependency-url]
+[![Coverage Status][coverage-img]][coverage-url]
 [![Gitter chat][gitter-img]][gitter-url]
-[![Stories in Ready][ready-img]][ready-url]
-[![Stories in Progress][progress-img]][progress-url]
+[![Stories in Ready][ready-img]][waffle]
```

Comment:
I kind of liked seeing the ready and in progress counts, the target url is the same but the counts are different.